contracts/multiply/DummyExchange.sol

```diff
@@ -81,9 +81,9 @@ contract DummyExchange {
   function swapDaiForToken(
     address asset,
     uint256 amount,
-    uint256 receiveAtLeast,
```

Comment:
this diff seems wrong, I did not download branch yet to see final files, but where names go?

Reply:
that's one of the solhint's warnings. if the variable is not used - no need to leave them to avoid ambiguity. i can leave the commented names

Reply:
ok I see, right in Dummy exchange do not use all inputs